### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by Lehren**

### CODE REVIEW

File: src/app/components/module-component/edit/module-edit.component.html
Comment (lukeelten) [inline src/app/components/module-component/edit/module-edit.component.html:325]: Despite that this will do the job, there is usually a better method for input validation than this tricky javascript hacks.

Comment (Lehren) [reply src/app/components/module-component/edit/module-edit.component.html:325]: we can figure that out Later™

**Comment by Lehren** [comment]
Once merged pls notify V.D Ham that his issues are fixed


(hopefully)

File: src/app/components/module-component/edit/module-edit.component.html
Comment (lukeelten) [reply src/app/components/module-component/edit/module-edit.component.html:325]: We can leave this to whoever is next

**Comment by lukeelten** [comment]
Is this pull request ready to merge?

**Review by lukeelten — APPROVED** [review]

**Pull request merged**
